.changeset/nervous-garlics-beam.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
+---
+
+Set action input default values from zod if FormData key is not present
```

Comment:
```suggestion
Fixes Astro Actions `input` validation when using `default` values with a form input.
```

Noticed the changeset described what code changes we made internally, instead of what was fixed _as a user._ Rephrased to describe the fix!